README.md

```diff
@@ -55,8 +55,22 @@ For developers:
 - [Translations](https://hosted.weblate.org/engage/mbin/)
 - [Contribution guidelines](CONTRIBUTING.md) - please read first, including before opening an issue!
 
+## Collaborators
+
+<!-- readme: collaborators -start -->
+<!-- readme: collaborators -end -->
+
+## Contributors
+
+<!-- readme: contributors -start -->
+<!-- readme: contributors -end -->
+
```

Comment:
I think this placement is a bit high up in the document. I don't know how it will look like, but I guess it will push everything underneath it way down, to the point where people do not expect there to be anything else. Could you provide a screenshot of how it looks? Or is it not visible until this hits main?